river/XwaylandView.zig

```diff
@@ -269,3 +280,12 @@ fn handleRequestFullscreen(listener: *wl.Listener(*wlr.XwaylandSurface), xwaylan
     self.view.pending.fullscreen = xwayland_surface.fullscreen;
     self.view.applyPending();
 }
+
+fn handleRequestMinimize(
+    listener: *wl.Listener(*wlr.XwaylandSurface.event.Minimize),
+    event: *wlr.XwaylandSurface.event.Minimize,
+) void {
+    const self = @fieldParentPtr(Self, "request_minimize", listener);
+    self.view.pending.minimized = event.minimize;
+    self.view.applyPending();
```

Comment:
This should just call `xwayland_surface.setMinimized()` directly. This is an implementation detail of xwayland and should not leak out in to the rest of the code.

This also needs a comment explaining why we need to do this.